CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to O-Neko
+
+*Disclaimer: We created O-Neko as a "lab day" project and had to deal with time limits during our regular
+work hours. This led to us neglecting some best practices in order to get the project to a usable state in an adequate
+time (it still took us almost two years from the idea to now). This trade off we had to make means the project is 
```

Comment:
"in an adequate time" sound strange to me. How about something like "in an acceptable amount of time"?